core/src/luigi-config/luigi-config.js

```diff
@@ -601,6 +598,16 @@ Promise.all(initPromises)
                 }
                 return [];
               });
+          window.clusterMicrofrontendNodesForNamespace =
+              cmfs
+                .filter(cmf => cmf.placement === 'namespace')
```

Comment:
Please note that after environment -> namespace renaming we kept the possibility to set the placement also to `environment` to be backwards compatible. In previous approach I see that it was accidentally changed (it should be `getUiEntities('clustermicrofrontends', namespace, [ 'namespace',	'environment'])`) and probably we should fix it here ;) 
```suggestion
                .filter(cmf => cmf.placement === 'namespace' || cmf.placement === 'environment')
```